local_server/local-server-feature-layer/README.md

```diff
@@ -1,36 +1,35 @@
-# Local Server Feature Layer
+# Local server feature layer
```

Comment:
looking at official doc for Local Server, it is a capitalized term, so we should ensure that's consistent
```suggestion
# Local Server feature layer
```